### ISSUE TITLE
Provide npm package

### ISSUE THREAD

**Issue by YZahringer** [creativetimofficial/material-dashboard-pro-angular#233]
It would be useful to provide the theme and updates through npm. 

A good example is Font Awesome Pro [https://fontawesome.com/how-to-use/use-with-node-js#pro](https://fontawesome.com/how-to-use/use-with-node-js#pro): To access to the npm packages, it is necessary to have a token provided with the paid Pro version.

**Comment by smunukuntla** [comment]
Yes this is something Creative Tim is lagging. I have purchased Creative Tim Angular Pro for one of my project and for other project which I'm currently working on, I'm using mdboostrap angular pro version using their npm module (which is super easy to integrate in plain angular project).

Dear Creative Tim, pls look into this.

**Comment by alexandru-paduraru** [comment]
@YZahringer @raskolnikov1041 @smunukuntla thank you for the feedback. We will have a look into npm package private repositories and see how we can create auth tokes for each of our customers. 

In the past, we used to deliver only plain HTML products based on Bootstrap. The updates there meant the replacement of 2 files. Now we expanded our frameworks so new things need to be implemented on our product update and product purchase flows in order to keep it simple for you guys. We will have this, no worries.

We will come with more details as soon as we start the implementation on our purchase flow. We have in plan to work in this beginning with 1 June 2018.

Best,
Alex

**Comment by vengaglobal** [comment]
Any updates on this?
